extensions/binding/org.eclipse.smarthome.binding.homematic/src/main/java/org/eclipse/smarthome/binding/homematic/handler/HomematicBridgeHandler.java

```diff
@@ -332,6 +338,35 @@ public void onDeviceLoaded(HmDevice device) {
         }
     }
 
+    @Override
+    public void onDutyCycleRatioUpdate(int dutyCycleRatio) {
+        ThingHandlerCallback callback = getCallback();
+        if (callback == null) {
+            logger.debug(
+                    "Duty Cycle ratio update is skipped for the homematic bridge '{}' because the handler is not initialized yet.",
+                    thing.getBridgeUID().getId());
```

Comment:
~~thing.getBridgeUID().getId()~~ -> `thing.getUID()`

You don't want to log the bridge's bridge, do you?

Reply:
Of course, I tend to forget that I already have the bridge when I see the field "thing"... 😅